requirements/test.txt

```diff
@@ -0,0 +1,3 @@
+mock
```

Comment:
Any chance we can replace `mock` with `pytest` monkeypatch so we can remove this dependency? It isn't clear to me what it's even used for.

Reply:
There was some error while importing torchvision, but I don't remember exactly. I think it can be replaced in case tests pass.